core/mondoo-slack-security.mql.yaml

```diff
@@ -0,0 +1,280 @@
+# Read more about the policy structure at https://mondoo.com/docs
+policies:
+  - uid: mondoo-slack-security
+    name: Slack Team Security by Mondoo
+    version: 1.1.0
+    tags:
+      mondoo.com/platform: slack-team,saas
+      mondoo.com/category: security
+    authors:
+      - name: Mondoo, Inc
+        email: hello@mondoo.com
+    docs:
+      desc: |
+        ## Overview
+
+        The Slack Team Security policy ensure best-practice settings for securing Slack Teams.
+
+        ### Prerequisites
+
+        To run this query pack, you will need access to the Slack API. To get a token, you need to create an App for the Slack workspace
+        and assign the appropriate permissions:
+
+        1. Sign in to [the Slack website](https://api.slack.com/apps/), and view **Your Apps**
+        2. Select **Create New App**
+        3. Select **From scratch**
+        4. Enter an "App Name" e.g. cnquery and select the workspace, then select **Create App**
+        5. In the section **Add features & functionality** select **Permissions**
+        6. Scroll to **Scopes** and then **User Token Scopes**
+
+          Note: Bots are very limited in their access; therefore we need to set the user scopes
+
+        7. Add the required permissions to **User Token Scopes**
+
+          | OAuth Scope  |
+          | ---- |
+          | [channels:read](https://api.slack.com/scopes/channels:read) |
+          | [groups:read](https://api.slack.com/scopes/groups:read) |
+          | [im:read](https://api.slack.com/scopes/im:read) |
+          | [mpim:read](https://api.slack.com/scopes/mpim:read) |
+          | [team:read](https://api.slack.com/scopes/team:read) |
+          | [usergroups:read](https://api.slack.com/scopes/usergroups:read) |
+          | [users:read](https://api.slack.com/scopes/users:read) |
+
+        8. Scroll up to **OAuth Tokens for Your Workspace** and select **Install to Workspace**
+        9. Copy the provided **User OAuth Token**, it will look like `xoxp-1234567890123-1234567890123-1234567890123-12345cea5ae0d3bed30dca43cb34c2d1`
+
+        ### Run policy
+
+        To run this policy against a Slack Workspace:
+
+        ```bash
+        export SLACK_TOKEN=xoxp-TOKEN
```

Comment:
We should also show the user how to use the CLI flag so they can use env vars or CLI flags in cnspec